mods/ra/rules/vehicles.yaml

```diff
@@ -733,6 +736,9 @@ QTNK:
 		RevealGeneratedShroud: False
 	RevealsShroud@GAPGEN:
 		Range: 4c0
+	RevealOnDeath:
```

Comment:
Oh yeah. No that's not indented at all.

I'd rather not overreach with this PR tho. Seem like a few things need to be sorted out to extend the feature over to a wider range of units as well. Perhaps just introduce this feature to a few exclusive units - aircrafts (excluding Chinook) and the demo truck? With those units the trait is specifically useful.